src/util/util.js

```diff
@@ -16,3 +16,10 @@ export const noop = () => {};
 export const noargs = (f) => () => f();
 
 export const sumUnderThreshold = (list, threshold) => list.reduce((acc, i) => acc + Math.min(i, threshold), 0);
+
+// To be able to mock it
+export const $location = {
```

Comment:
Could we name this something other than `$location`? The `$` makes me think of a Vue global. Ideas:

- `mockableLocation`. Not a pretty name, but it makes it clear why it's being used over `window.location`.
- `appLocation`
- We could also add `origin` to the container in `createContainer()`, then replace it in the test. `origin` could be a getter on the container. The container is a common destination for global-y things or things that need to be mocked.

Reply:
I like the 3rd idea. But I have added `location` instead of `origin` in the container as there is another place where I need to mock location.assign